### PR TITLE
add VSR-I chassis type to support network functions based on the isa card 

### PR DESCRIPTION
**Pull request by mairklone**
tested on 21.10.R3 and 22.10.R1

### CODE REVIEW

File: sros/docker/launch.py [hunk 1]
```diff
@@ -251,6 +251,20 @@ def line_card_config(chassis, card, mda, integrated=False, card_type=None):
...
     },
+    "vsr-i": {
+        "deployment_model": "integrated",
+        "min_ram": 8,  # minimum RAM requirements
+        "max_nics": 20,
+        "timos_line": "chassis=VSR-I slot=A card=iom-v mda/1=m20-v mda/2=isa-tunnel-v",
+        "card_config": """/configure card 1 card-type iom-v
+        /configure card 1 mda 1 mda-type m20-v
+        /configure card 1 mda 2 mda-type isa-tunnel-v
+        """,
```
Comment (hellt) [inline sros/docker/launch.py:262]: Hi @mairklone 
would you be able to adapt this piece to using the line_card_config function
https://github.com/hellt/vrnetlab/blob/master/sros/docker/launch.py#L246

Comment (mairklone) [reply sros/docker/launch.py:262]: Hi @hellt 
the only thing that I am seeing is that the function so far only allows me to add one mda:
https://github.com/hellt/vrnetlab/blob/0815321fab3d35dfabe4ecf25d7c91dbc8013c7b/sros/docker/launch.py#L49
In my humble opinion for this type of chassis being functional both mdas need to be provision, what do you think?

Comment (hellt) [reply sros/docker/launch.py:262]: ah, okay, we will have to enhance it then, but not in this PR
this looks good, thanks

**Comment by mairklone** [comment]
Just bringing my two cents, I have realized that my colleagues and myself
were using VSR-I a lot specially when we need to work with network
functions based on the ISA card normally on KVM but our labs were
completely manually created now available on vrnetlab. By the other hand
actually I was not aware about this function looks like is new and I think
that is a really good Idea.
I am the guy that just need to say thanks to you for all your great stuff!!!

BUILD-CONQUER-REPEAT

**Pull request merged**